tests/runtime/probe

```diff
@@ -100,18 +100,28 @@ WILL_FAIL
 NAME kprobe_offset_module
 RUN {{BPFTRACE}} -e 'kprobe:nft_trans_alloc_gfp+0x5 { printf("hit\n"); exit(); }' -c '/usr/sbin/nft add table bpftrace'
 EXPECT hit
+ARCH x86_64|aarch64
 TIMEOUT 5
 REQUIRES lsmod | grep '^nf_tables'
 REQUIRES /usr/sbin/nft --help
-AFTER nft delete table bpftrace
+CLEANUP nft delete table bpftrace
+
+# Local entry point to nft_trans_alloc_gfp is located at offset of 8 bytes in ppc64.
+NAME kprobe_offset_module
+RUN {{BPFTRACE}} -e 'kprobe:nft_trans_alloc_gfp+0x8 { printf("hit\n"); exit(); }' -c '/usr/sbin/nft add table bpftrace'
+EXPECT hit
+ARCH ppc64|ppc64le
+TIMEOUT 5
+REQUIRES lsmod | grep '^nf_tables'
+REQUIRES /usr/sbin/nft --help
+CLEANUP nft delete table bpftrace
 
 NAME kprobe_offset_module_error
 RUN {{BPFTRACE}} -e 'kprobe:nft_trans_alloc_gfp+0x1 { printf("hit\n"); exit(); }' -c '/usr/sbin/nft add table bpftrace'
 EXPECT Possible attachment attempt in the middle of an instruction, try a different offset.
 TIMEOUT 5
 REQUIRES lsmod | grep '^nf_tables'
 REQUIRES /usr/sbin/nft --help
-AFTER nft delete table bpftrace
```

Comment:
Should be replaced with a CLEANUP here, right?

Reply:
Here we expect the failure due to incorrect offset in between instruction. I was not sure of CLEANUP error getting reported in a test where we expect other reason.

Reply:
Ah, that is a good point. How about deleting the above `-c '/usr/sbin/nft add table bpftrace'` in the test just so we're certain there's nothing to clean up